src/js/jsx/sections/layers/LayerFace.jsx

```diff
@@ -229,13 +237,14 @@ define(function (require, exports, module) {
          * @param {SyntheticEvent} event
          */
         _handleLayerEdit: function (event) {
+            event.stopPropagation();
```

Comment:
Is there a particular reason why you moved these `stopPropagation` calls? Or does it just seem like a good pattern?